### ISSUE TITLE
[regressions] lost `?>` in program node

### ISSUE THREAD

**Issue by evilebottnawi** [glayzzle/php-parser#176]
Input:
```
<?php
__LINE__
?>
```

Reproducible:
```
const ast = parser.parseCode(text);

console.log(ast.loc.source);
```

`ast.loc.source` doesn't contain `?>`, looks like regressions because in `3.0.0-alpha2` this token exists in source

**Comment by ichiriac** [comment]
yes, introduced by ignoring the close tag https://github.com/glayzzle/php-parser/issues/168 - I'll fix it but it could be a good practice to remove the last `?>` if there is no output after, in order to avoid extra spacing errors that could trigger a bad echo

**Comment by ichiriac** [comment]
fixed under 3.0.0-prerelease.3
